iree/compiler/Conversion/LinalgToSPIRV/Passes.cpp

```diff
@@ -103,9 +103,7 @@ static void addLinalgToSPIRVPasses(OpPassManager &pm,
   //===--------------------------------------------------------------------===//
   pm.addPass(createSplitDispatchFunctionPass());
   pm.addPass(createLinalgTileAndFusePass(options));
-  if (options.useVectorPass) {
```

Comment:
also remove this option?

Reply:
Good catch...and also clean the tests. Thanks!